### ISSUE TITLE
No support for old mods?

### ISSUE THREAD

**Issue by TedThompson** [yquake2/yquake2#874]
Tried to launch an old SP mod I made 20 years ago, game just crashes to desktop (windows) when I try to start via Menu->difficulty. 

DLL is gamex86.dll, renaming to game.dll gave the same results. 

Is this normal because its compiled for the original client? I searched but didn't find an answer.

**Comment by maraakate** [comment]
Hi Ted, you have to use the 32-bit version of Yamagi Quake 2 in order to use your old mod.  It will not work with the 64-bit version.

**Comment by TedThompson** [comment]
Theres only 2 executables in is the 8.10 package, yquake2.exe and quake2.exe - where does one obtain the "32-bit version"?

**Comment by Yamagi** [comment]
The official Windows release is 32 bit for mod compatibility. Old mods should still work, this looks like a bug. Can you attach the log file of a failed attempt? It's stdout.txt under C:\User\$Username\Document\YamagiQ2.

**Comment by DanielGibson** [comment]
Was the mod for the last official Quake2 patch (3.20, I think)?
I can imagine that they broke the game API at some point..

**Comment by TedThompson** [comment]
So I did some more tests. 

I downloaded "Generations" from modDB as it appeared to have been last compiled back in 1998

I then made a separate quake2 install using gilrs from steam and overwrighting them with last ID update package, replacing quake.exe

I ran "Vanilla Quake2" to verify it worked. 

I put Generations and my old mod in BOTH installs. 

I attempted to run both mods in both vanilla and Yamagi. 

Generations worked in both*

My mod crashed in both. (My old mod is either corrupt or not compiled for 3.20)

*I'll make a separate ticket for my experience running the Generations mod in Yagami.
